src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+public final class TestStringHelper {
```

Comment:
Okay option 3 then. To me `StringHelperTestHelper` sounds like it's a helper that's used only in `StringHelperTest`, when in fact it's used in many tests.